imap_processing/cli.py

```diff
@@ -621,24 +630,31 @@ def do_processing(self, dependencies):
         elif self.data_level == "l1b":
             data_dict = {}
             for dependency in dependencies:
-                dataset = load_cdf(dependency, to_datetime=True)
+                dataset = load_cdf(dependency, to_datetime=True)  # type: ignore[arg-type]
+                # Todo Change
+                # Argument "to_datetime" to "load_cdf" has incompatible
+                # type "bool"; expected "dict[Any, Any]"
```

Comment:
```suggestion
                dataset = load_cdf(dependency, to_datetime=True)  # type: ignore[arg-type]
```

I'm happy with just keeping the inline `ignore` statement here